src/gui/EffectSelectDialog.cpp

```diff
@@ -173,25 +175,64 @@ void EffectSelectDialog::rowChanged( const QModelIndex & _idx,
 	{
 		m_currentSelection = m_effectKeys[m_model.mapToSource( _idx ).row()];
 	}
-	if( m_currentSelection.desc && m_currentSelection.desc->subPluginFeatures )
+    if( m_currentSelection.desc )
 	{
 		m_descriptionWidget = new QWidget;
-		QVBoxLayout * l = new QVBoxLayout( m_descriptionWidget );
-		l->setMargin( 4 );
-		l->setSpacing( 0 );
-
-		ui->scrollArea->setWidget( m_descriptionWidget );
 
-		m_currentSelection.desc->subPluginFeatures->
-			fillDescriptionWidget( m_descriptionWidget, &m_currentSelection );
-		foreach( QWidget * w, m_descriptionWidget->findChildren<QWidget *>() )
-		{
-			if( w->parent() == m_descriptionWidget )
-			{
-				l->addWidget( w );
-			}
-		}
-		l->setSizeConstraint( QLayout::SetFixedSize );
+        QHBoxLayout *hbox = new QHBoxLayout( m_descriptionWidget );
+
+        Plugin::Descriptor const & descriptor = *( m_currentSelection.desc );
+
+        if ( descriptor.logo )
+        {
+            QLabel *logoLabel = new QLabel( m_descriptionWidget );
+            logoLabel->setPixmap( descriptor.logo->pixmap() );
+            logoLabel->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Minimum);
+
+            hbox->addWidget( logoLabel );
+            hbox->setAlignment( logoLabel, Qt::AlignTop);
+        }
+
+        QWidget *textualInfoWidget = new QWidget( m_descriptionWidget );
+
+        hbox->addWidget(textualInfoWidget);
+
+        QVBoxLayout * textWidgetLayout = new QVBoxLayout( textualInfoWidget);
```

Comment:
Thanks! I think in times of IDEs with autocompletion variable names like `l` should be made a thing of the past. :)